Kitodo/src/main/java/de/sub/goobi/forms/ProjekteForm.java

```diff
@@ -800,4 +843,118 @@ public int getItemId() {
         return this.itemId;
     }
 
+    /**
+     * Get the XML configuration string.
+     *
+     * @return the XML configuration string
+     */
+    public String getXMLConfiguration() {
+        return this.xmlConfigurationString;
+    }
+
+    /**
+     * Set the XML configuration string.
+     *
+     * @param configuration
+     *            the XML configuration string
+     */
+    public void setXMLConfiguration(String configuration) {
+        this.xmlConfigurationString = configuration;
+    }
+
+    /**
+     * Load the content of the XML configuration file denoted by 'configurationFile'
+     * as a String.
+     */
+    public void loadXMLConfiguration(String configurationName) {
+        try {
+            currentConfigurationFile = configurationFiles.get(configurationName);
+            XMLConfiguration currentConfiguration = new XMLConfiguration(
+                    ConfigCore.getKitodoConfigDirectory() + currentConfigurationFile);
+            StringWriter stringWriter = new StringWriter();
```

Comment:
Opening a output object without closing it after usage. Maybe you can use for this object a [try-with-resource statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html).